worlds/generic/docs/setup_en.md

```diff
@@ -17,19 +17,22 @@ The most recent public release of Archipelago can be found on the GitHub Release
 Run the exe file, and after accepting the license agreement you will be asked which components you would like to
 install.
 
-The generator allows you to generate multiworld games on your computer. The ROM setups are required if anyone in the
-game that you generate wants to play any of those games as they are needed to generate the relevant patch files. If you
-do not own the game, uncheck the relevant box. If you gain the game later, the installer can be run again to install and
-set up new components.
+Archipelago installations are automatically bundled with some programs. These include a launcher, a generator, a
+server and some clients.
 
-The server will allow you to host the multiworld on your machine. Hosting on your machine requires forwarding the port
+- The launcher lets you quickly access Archipelago's different components and programs. It is found under the name 
+  `ArchipelagoLauncher` and can be found in the main directory of your Archipelago installation.
+
+- The generator allows you to generate multiworld games on your computer. Please refer to the 'Generating a game'
+  section of this guide for more information about it.
+
+- The server will allow you to host the multiworld on your machine. Hosting on your machine requires forwarding the port
 you are hosting on. The default port for Archipelago is `38281`. If you are unsure how to do this there are plenty of
 other guides on the internet that will be more suited to your hardware.
 
-The `Clients` are what are used to connect your game to the multiworld. If the game you plan to play is available
-here, go ahead and install its client as well. If the game you choose to play is supported by Archipelago but not listed
-in the installation, check the setup guide for that game. Installing a client for a ROM based game requires you to have
-a legally obtained ROM for that game as well.
+- The `Clients` are what are used to connect your game to the multiworld. Some games use a client that is automatically
+installed with an Archipelago installation. You can access those clients via the launcher or by navigating
+to your Archipelago installation. 
```

Comment:
I would add a note that if they don't see a client, they will need to check the game page to learn how to set up the necessary connection to the multiworld

Reply:
I chose not to because I believe every player should check the respective setup guide.

Reply:
Maybe a link could be added in the connecting section